lib/winapi/sysinfoapi.h

```diff
@@ -0,0 +1,49 @@
+#ifndef __SYSINFOAPI_H__
+#define __SYSINFOAPI_H__
+
+#include <minwinbase.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+void GetSystemTimePreciseAsFileTime (LPFILETIME lpSystemTimeAsFileTime);
+DWORD GetTickCount (void);
+
+// Unprovided fields are intentionally disabled to catch code trying to access them
+typedef struct _SYSTEM_INFO
+{
+    union
+    {
+        //DWORD dwOemId;
```

Comment:
Why dead code?

Reply:
Nevermind, there's a comment above. I don't fully agree with this approach though (offsets of fields shouldn't change on Xbox).

Reply:
I'm not fully satisfied with my solution, either. Ideally, there would be an attribute to mark the fields so that the user gets an error or a warning, but there doesn't seem to be such a thing.